cg/store/models.py

```diff
@@ -608,6 +608,8 @@ def sequencing_qc(self) -> bool:
         if self.priority == Priority.express:
             one_half_of_target_reads = application.target_reads / 2
             return self.reads >= one_half_of_target_reads
+        if self.application_version.application.prep_category == "rml":
```

Comment:
Should we replace the "rml" with the constant PrepCategory. READY_MADE_LIBRARY.value ?

Reply:
Did not know we had one! Thanks 👍